Adafruit_MQTT.cpp

```diff
@@ -434,25 +437,25 @@ void Adafruit_MQTT::processPackets(int16_t timeout) {
     if (sub) {
       // Serial.println("**** sub packet received");
       if (sub->callback_uint32t != NULL) {
-        // huh lets do the callback in integer mode
-        uint32_t data = 0;
-        data = atoi((char *)sub->lastread);
-        // Serial.print("*** calling int callback with : ");
-        // Serial.println(data);
-        sub->callback_uint32t(data);
-      } else if (sub->callback_double != NULL) {
-        // huh lets do the callback in doublefloat mode
-        double data = 0;
-        data = atof((char *)sub->lastread);
-        // Serial.print("*** calling double callback with : ");
-        // Serial.println(data);
-        sub->callback_double(data);
-      } else if (sub->callback_buffer != NULL) {
-        // huh lets do the callback in buffer mode
-        // Serial.print("*** calling buffer callback with : ");
-        // Serial.println((char *)sub->lastread);
-        sub->callback_buffer((char *)sub->lastread, sub->datalen);
-      } else if (sub->callback_io != NULL) {
+	// huh lets do the callback in integer mode
+	uint32_t data = 0;
+	data = atoi((char *)sub->lastread);
+	//Serial.print("*** calling int callback with : "); Serial.println(data);
```

Comment:
Perhaps use `DEBUG_PRINT` and not Serial.print?   This way we can keep these debug messages?

Reply:
This is commented out code that existed before I refactored - can probably remove it entirely

Reply:
@Fapiko Could you remove that entirely?